fe1-web/src/core/network/ingestion/MessageReducer.ts

```diff
@@ -33,7 +33,7 @@ const messagesSlice = createSlice({
   reducers: {
     // Add a Message to the list of known Messages
     addMessages: {
-      prepare(messages: ExtendedMessageState | ExtendedMessageState[]): any {
```

Comment:
I tried to add `prepare` methods in most places that take care of the conversion of class instances to the respective state values